### ISSUE TITLE
Import Error with PyGObject

### ISSUE THREAD

**Issue by danyeaw** [msys2/MINGW-packages#6368]
I noticed my builds started to fail in Windows. I updated all the MSYS2 packages locally to try to see what is going on, and PyGObject is giving an ImportError:

```
$ python
Python 3.8.2 (default, Apr  9 2020, 13:17:39)  [GCC 9.3.0 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import gi
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:/tools/msys64/mingw64/lib/python3.8/site-packages/gi/__init__.py", line 42, in <module>
    from . import _gi
ImportError: DLL load failed while importing _gi: The specified module could not be found.
```
I have mingw-w64-x86_64-gtk3, mingw-w64-x86_64-python, and mingw-w64-x86_64-python-gobject installed.

**Comment by danyeaw** [comment]
@lazka I hope you are staying safe! Any thoughts on what the root cause DLL load error could be?

**Comment by lazka** [comment]
This is fixed by 3f880aaba91a3d9 but needs to get into the repo

**Comment by danyeaw** [comment]
@lazka Thanks! I'll go ahead and close this issue then 😄
